### PR TITLE
Automation, commenting, localization and bugfixes

### PR DESCRIPTION
**Pull request by thalissa**
Added willpower rerolls dealing willpower damage and rouse checks increasing hunger (both toggle-able in the settings per-user), as well as added better commenting/organization to the diceroller.

### CODE REVIEW

**Comment by Rayji96** [comment]
I updated the linter. Pull from rji/dev (the dev branch; maybe I should rename it lol) to get the liner running :)

File: module/actor/roll-dice.js [hunk 1]
```diff
@@ -2,17 +2,28 @@
...
     hungerDice = Math.min(actor.data.data.hunger.value, numDice)
   } else {
     hungerDice = 0
   }
+
+  // Roll defining and evaluating
   const dice = numDice - hungerDice
   const roll = new Roll(dice + 'dvcs>5 + ' + hungerDice + 'dhcs>5', actor.data.data)
-  await roll.evaluate()
+  roll.evaluate()
```
Comment (Rayji96) [inline module/actor/roll-dice.js:24]: Does this work?

**Comment by Rayji96** [comment]
Can you rebase off rji/dev? Github seems to be struggling with the duplicate commits

File: lang/en.json [hunk 2]
```diff
@@ -12,9 +12,11 @@
     "VTM5E.Health": "Health",
     "VTM5E.Willpower": "Willpower",
     "VTM5E.WillpowerReroll": "Willpower Reroll",
+    "VTM5E.WillpowerFull": "Willpower tracker is already full, cannot increase it further.",
     "VTM5E.Exp": "Exp",
     "VTM5E.Humanity": "Humanity",
     "VTM5E.Hunger": "Hunger",
+    "VTM5E.HungerFull": "Hunger tracker is already full, cannot increase it further.",
```
[the remark this answers is not in the document]
Comment (thalissa) [reply lang/en.json:19]: I can update these two in order to reflect those preferences, yeah! I just went with something that seemed the most obvious, but I do agree about the not immersive aspect of it. As for Google Translated placeholders, I can do that as well!

File: css/vtm5e.css [hunk 3]
```diff
@@ -389,11 +348,8 @@
 .vtm5e .tabs .item {
   line-height: 40px;
   font-weight: bold;
-}
-
-.vtm5e .tabs .item.active {
```
[the remark this answers is not in the document]
Comment (thalissa) [reply css/vtm5e.css:394]: The Linter yelled at me which is why I removed it, however the styling is still there on line 190!

File: module/actor/roll-dice.js (hunk 1 above)
Comment (thalissa) [reply module/actor/roll-dice.js:24]: Yes it does! In v9 roll.evaluate() is becoming asynchronous and thus the await isn't necessary, is what I gathered.

**Comment by thalissa** [comment]
Can do!

> Can you rebase off rji/dev? Github seems to be struggling with the duplicate commits

And can do while I'm fixing the translations!

**Comment by thalissa** [comment]
There we go! Should be all fixed

**Pull request merged**